platforms/unix/plugins/SqueakSSL/openssl_overlay.h

```diff
@@ -0,0 +1,301 @@
+  _C(sqo_SSL_accept = (int (*)(SSL *ssl)) _sqo_find("SSL_accept"));
+  _C(sqo_SSL_connect = (int (*)(SSL *ssl)) _sqo_find("SSL_connect"));
+  _C(sqo_SSL_free = (void (*)(SSL *ssl)) _sqo_find("SSL_free"));
+  _C(sqo_SSL_ctrl = (long (*)(SSL *ssl, int cmd, long larg, void *parg)) _sqo_find("SSL_ctrl"));
+  _C(sqo_SSL_get_error = (int (*)(const SSL *s, int ret_code)) _sqo_find("SSL_get_error"));
+  _C(sqo_SSL_get_peer_certificate = (X509 *(*)(const SSL *s)) _sqo_find("SSL_get_peer_certificate"));
+  _C(sqo_SSL_get_verify_result = (long (*)(const SSL *ssl)) _sqo_find("SSL_get_verify_result"));
+  _C(sqo_SSL_library_init = (int (*)(void)) _sqo_find("SSL_library_init"));
+  _C(sqo_SSL_load_error_strings = (void (*)(void)) _sqo_find("SSL_load_error_strings"));
+  _C(sqo_SSL_new = (SSL *(*)(SSL_CTX *ctx)) _sqo_find("SSL_new"));
+  _C(sqo_SSL_read = (int (*)(SSL *ssl, void *buf, int num)) _sqo_find("SSL_read"));
+  _C(sqo_SSL_set_accept_state = (void (*)(SSL *s)) _sqo_find("SSL_set_accept_state"));
+  _C(sqo_SSL_set_bio = (void (*)(SSL *s, BIO *rbio, BIO *wbio)) _sqo_find("SSL_set_bio"));
+  _C(sqo_SSL_set_connect_state = (void (*)(SSL *s)) _sqo_find("SSL_set_connect_state"));
+  _C(sqo_SSL_write = (int (*)(SSL *ssl, const void *buf, int num)) _sqo_find("SSL_write"));
+  _C(sqo_X509_NAME_get_text_by_NID = (int (*)(X509_NAME *name, int nid, char *buf, int len)) _sqo_find("X509_NAME_get_text_by_NID"));
+  _C(sqo_X509_get_subject_name = (X509_NAME *(*)(X509 *a)) _sqo_find("X509_get_subject_name"));
+  _C(sqo_X509_get_ext_d2i = (void *(*)(X509 *x, int nid, int *crit, int *idx)) _sqo_find("X509_get_ext_d2i"));
+  _C(sqo_X509_free = (void (*)(X509 *)) _sqo_find("X509_free"));
+  sqo_X509_check_ip_asc = (int (*)(X509 *x, const char *ipasc, unsigned int flags)) _sqo_find("X509_check_ip_asc");
+  sqo_X509_check_host = (int (*)(X509 *x, const char *chk, size_t chklen, unsigned int flags, char **peername)) _sqo_find("X509_check_host");
+
+#if OPENSSL_VERSION_NUMBER >= 0x10000000L
+  _C(sqo_sk_new_null = (_STACK *(*)(void)) _sqo_find("sk_new_null"));
+  _C(sqo_sk_push = (int (*)(_STACK *st, void *data)) _sqo_find("sk_push"));
+  _C(sqo_sk_free = (void (*)(_STACK *st)) _sqo_find("sk_free"));
+  _C(sqo_sk_value = (void *(*)(const _STACK *st, int i)) _sqo_find("sk_value"));
+  _C(sqo_sk_num = (int (*)(const _STACK *st)) _sqo_find("sk_num"));
+  _C(sqo_sk_pop_free = (void (*)(_STACK *st, void (*func) (void *))) _sqo_find("sk_pop_free"));
+#else
+  _C(sqo_sk_new_null = (STACK *(*)(void)) _sqo_find("sk_new_null"));
+  _C(sqo_sk_push = (int (*)(STACK *st, char *data)) _sqo_find("sk_push"));
+  _C(sqo_sk_free = (void (*)(STACK *st)) _sqo_find("sk_free"));
+  _C(sqo_sk_value = (char *(*)(STACK *st, int i)) _sqo_find("sk_value"));
+  _C(sqo_sk_num = (int (*)(STACK *st)) _sqo_find("sk_num"));
+  _C(sqo_sk_pop_free = (void (*)(STACK *st, void (*func) (void *))) _sqo_find("sk_pop_free"));
+#endif // OPENSSL_VERSION_NUMBER >= 0x10000000L
+  return true;
```

Comment:
Can't this be in the previous `#if OPENSSL_VERSION_NUMBER >= 0x10000000L` in line 282?